Dialog/TilesetEditDialog.cpp

```diff
@@ -1184,6 +1230,31 @@ void TilesetEditDialog::on_pushButton_ImportPalette_clicked()
             tilesetEditParams->newTileset->SetColor(SelectedPaletteId, j, newcolor.rgba());
         }
     }
+
+    else if (selectedfilter.compare("Raw Binary palette (*.bin)") == 0)
+    {
+         QByteArray tmppalettedata;
+         QFile palbinfile(qFilePath);
+         if(!palbinfile.open(QIODevice::ReadOnly))
+         {
+             QMessageBox::critical(this, QString("Error"), QString("Cannot open file!"));
```

Comment:
Let's also include the error reason: https://doc.qt.io/qt-5/qfiledevice.html#error

Reply:
:heavy_check_mark: